pkg/agent/flowexporter/connections/conntrack.go

```diff
@@ -26,11 +26,12 @@ import (
 )
 
 // InitializeConnTrackDumper initializes the ConnTrackDumper interface for different OS and datapath types.
-func InitializeConnTrackDumper(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet, ovsctlClient ovsctl.OVSCtlClient, ovsDatapathType string) ConnTrackDumper {
+func InitializeConnTrackDumper(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet, ovsDatapathType string) ConnTrackDumper {
 	var connTrackDumper ConnTrackDumper
 	if ovsDatapathType == ovsconfig.OVSDatapathSystem {
 		connTrackDumper = NewConnTrackSystem(nodeConfig, serviceCIDR)
 	} else if ovsDatapathType == ovsconfig.OVSDatapathNetdev {
+		ovsctlClient := ovsctl.NewClient(nodeConfig.OVSBridge)
```

Comment:
Can we define this directly in the function `NewConnTrackOvsAppCtl` instead of passing this as an argument?

Reply:
I find that we need this argument to pass mockOVSCtlClient in unit test. Maybe we can discuss it more.